src/setup_payload/AdditionalDataPayloadGenerator.h

```diff
@@ -69,28 +69,28 @@ class AdditionalDataPayloadGenerator
      *                              TLVBackingStore
      *
      */
-    CHIP_ERROR generateAdditionalDataPayload(uint16_t lifetimeCounter, const char * serialNumberBuffer,
-                                             size_t serialNumberBufferSize, chip::System::PacketBufferHandle & bufferHandle,
+    CHIP_ERROR generateAdditionalDataPayload(uint16_t lifetimeCounter, MutableByteSpan uniqueId,
```

Comment:
```suggestion
    CHIP_ERROR generateAdditionalDataPayload(uint16_t lifetimeCounter, const ByteSpan & uniqueId,
```
This function does not mutate that buffer, right?